packages/client/screens/WalletOverviewScreen.tsx

```diff
@@ -52,14 +55,18 @@ export default function WalletOverviewScreen({ route, navigation }: Props) {
                             </Text>
                         </VStack>
                         <VStack>
-                            <FontAwesomeIcon icon={faEthereum} style={styles.ethereumIcon} size={40} />
+                            <FontAwesomeIcon
+                                icon={currencyIcon ? currencyIcon : faBitcoin}
```

Comment:
Isn't the `faBitcoin` redundant here since the icon is being determined by the `currencyTypeToIcon` function? Can be replaced with undefined.

```suggestion
                                icon={currencyIcon ? currencyIcon : undefined}
```